packages/@lwc/engine-core/src/framework/mutation-logger.ts

```diff
@@ -50,6 +50,16 @@ function toPrettyMemberNotation(parent: PropertyKey | undefined, child: Property
     }
 }
 
+function safelyCallGetter(target: any, key: PropertyKey) {
```

Comment:
Now "getter that throws" is indistinguishable from "getter that returns `undefined`". Does that matter?

What happens if the getter that _is_ used throws?

Reply:
It doesn't matter. The whole point of this routine is just to deep-traverse and object and set up a mapping between the objects and a pretty-printable name for it (`foo.bar.baz`). `undefined` will be ignored, same as throwing.

This means that the tracking is not perfect, but it wouldn't be anyway if e.g. the getter returned a random result every time.